frontend/src/metabase/static-viz/components/ComboChart/ComboChart.stories.tsx

```diff
@@ -408,13 +408,6 @@ ComboHistogram.args = {
   renderingContext,
 };
 
-export const ComboBarsBecomeStackedInStaticViz38827 = Template.bind({});
```

Comment:
I do not remember what this was. Do the new cases cover this old case?

Reply:
It tested that combo charts can't have stacked bars but now we are implementing this